x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/notification/jira/JiraAccount.java

```diff
@@ -64,7 +63,6 @@ public JiraAccount(String name, Settings settings, HttpClient httpClient) {
         this.httpClient = httpClient;
         this.name = name;
         String url = getSetting(name, URL_SETTING, settings, SECURE_URL_SETTING);
-        ESLoggerFactory.getLogger(getClass()).error("THE URL WAS [{}]", url);
```

Comment:
This seems like it was left here by mistake.

Reply:
I think so too, but maybe @spinscale can confirm this.

Reply:
clearly a mistake